### ISSUE TITLE
Cannot npm install on Windows

### ISSUE THREAD

**Issue by mateodelnorte** [GoogleChrome/accessibility-developer-tools#317]
```
C:\development\electron-react-boilerplate>npm i accessibility-developer-tools
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files (x86)\\Nodist\\v-x64\\6.9.1\\node.exe" "C:\\Program Files (x86)\\Nodist\\npmv\\3.10.9\\bin\\npm-cli.js" "i" "accessibility-developer-tools"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.9

npm ERR! shasum check failed for C:\Users\Waltem7\AppData\Local\Temp\npm-6864-c93f02db\registry.npmjs.org\accessibility-developer-tools\-\accessibility-developer-tools-2.11.0.tgz
npm ERR! Expected: 2b669c802671672aed5c61c42a782082a2fc7d2c
npm ERR! Actual:   ba1966ded6484349ff3ed9f55708882ad5031690
npm ERR! From:     https://registry.npmjs.org/accessibility-developer-tools/-/accessibility-developer-tools-2.11.0.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\development\electron-react-boilerplate\npm-debug.log
```
